dags/blocksec_plugin/contract_interaction_operator.py

```diff
@@ -10,7 +11,7 @@ class ContractInteractionOperator(BaseOperator):
     Executes a generalized contract interaction
     """
 
-    template_fields = ['contract_address', 'function', 'args', 'abi_json']
+    template_fields = ['contract_address', 'function', 'args', 'abi_json', 'transaction_hash']
 
     @apply_defaults
     def __init__(self,
```

Comment:
add `confirmations=3` as the default, add `self.confirmations = confirmations` into the init method like is done in the EthereumTransactionSensor's `__init__` method